api/clients/v2/payload_retriever.go

```diff
@@ -17,29 +17,29 @@ import (
 	gethcommon "github.com/ethereum/go-ethereum/common"
 )
 
-// EigenDAClient provides the ability to get payloads from the relay subsystem, and to send new payloads to the disperser.
+// PayloadRetriever provides the ability to get payloads from the relay subsystem.
```

Comment:
Will we use the same struct later to also do distributed retrievals? Or will that be a different one? PayloadRetriever in my mind should be able to do both, in case the relays are all censoring.

Reply:
I agree, distributed retrievals should be in this same struct.

I will amend to comment to declare the future intention

Reply:
added [4fa78a99](https://github.com/Layr-Labs/eigenda/pull/1155/commits/4fa78a999dc01392d9b708090d913c72230ad760)